tests/functional/t/typedDict.py

```diff
@@ -1,18 +1,34 @@
 """Test typing.TypedDict"""
-# pylint: disable=invalid-name,missing-class-docstring
+# pylint: disable=invalid-name,missing-class-docstring,pointless-statement
 import typing
 from typing import TypedDict
 
 
 class CustomTD(TypedDict):
     var: int
 
+
 class CustomTD2(typing.TypedDict, total=False):
     var2: str
 
+
 class CustomTD3(CustomTD2):
     var3: int
 
+
 CustomTD4 = TypedDict("CustomTD4", var4=bool)
 
 CustomTD5 = TypedDict("CustomTD5", {"var5": bool})
+
+# Checks that class used in a subscript supports subscription
+# (i.e. defines __class_getitem__ method).
```

Comment:
Did you mean `__getitem__`? AFAIK `__class_getitem__` is mostly used for typing purposes at the moment.
I would recommend to just remove it as it's not really necessary.